src/components/Form/FormArea/index.style.ts

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 19px;
+`
+
+const LabelContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const Label = styled.span`
```

Comment:
```suggestion
const Label = styled.label`
```
컴포넌트 이름은 Label인데  태그는 span이네요.